### DISCUSSION TITLE
No Disk Space Left error massage

### DISCUSSION THREAD

**Discussion by iguys** [nextcloud/all-in-one#3849] [Questions] [answered]
Hi,
I recently installed nextcloud aio on my Debian server with Nginx proxy manager installed on separate docker container outside that Debian server. I followed instructions on reverse proxy and everything working and I am able to upload files but when ever I try to open or create office files , it’s giving me error “No disk space left on sever close” . 
![IMG_6994](https://github.com/nextcloud/all-in-one/assets/96700289/75dfbd30-fe8b-4caa-8df1-60d561ae9d29)

**Comment by szaimen** [comment, marked as answer]
> No disk space left on sever
